src/vs/vscode.d.ts

```diff
@@ -10672,6 +10672,12 @@ declare module 'vscode' {
 		 * input with this value when appropriate.
 		 */
 		commitTemplate?: string;
+		
+		/**
+		 * The optional human-readable name of the source control
+		 * that replaces the default name if specified.
+		 */
+		visibleName?: string;
```

Comment:
Introducing new API needs to go through the proposal process: https://github.com/Microsoft/vscode/wiki/Extension-API-process.

Let's move this over to `vscode.proposed.d.ts` and open a new issue arguing for the existence of this new field in the API.

Reply:
And how about we simply call it `name`?